doc/guides/kconfig/index.rst

```diff
@@ -837,7 +837,7 @@ Device Tree Related Functions
            'm' or 'M'  divide by 1,048,576 (1 << 20)
            'g' or 'G'  divide by 1,073,741,824 (1 << 30)
 
-  dt_str_val(kconf, _, name):
+  dt_chosen_label(kconf, _, name):
```

Comment:
Would be nice to have documentation for the other ones.

Maybe this page could just refer to `scripts/kconfig/kconfigfunctions.py` for some or all of the functions, to avoid having to duplicate the documentation from the docstrings.

Reply:
Yeah, the doc commit I wanted to wait on til after all the code was good.  Will see if there's some way of just referencing the docstrings.